presto-druid/src/main/java/com/facebook/presto/druid/DruidClient.java

```diff
@@ -183,4 +185,44 @@ public InputStream handle(Request request, com.facebook.airlift.http.client.Resp
             }
         }
     }
+    public static class DruidRequestBody
+    {
+        private String query;
+        private String resultFormat;
+        private boolean queryHeader;
+        @JsonCreator
+        public DruidRequestBody(
+                @JsonProperty("query") String query,
+                @JsonProperty("resultFormat") String resultFormat,
+                @JsonProperty("queryHeader") boolean queryHeader)
+        {
+            requireNonNull(query);
+            this.query = query;
+            this.resultFormat = resultFormat;
```

Comment:
this. resultFormat = requireNonNull(resultFormat);

Reply:
done